monai/transforms/post/array.py

```diff
@@ -248,3 +248,60 @@ def __call__(self, img):
             output = img
 
         return output
+
+
+class Contour(Transform):
```

Comment:
I feel the design needs to be improved, please see kornia's abstraction of building kernels, setting up the padding, etc https://github.com/kornia/kornia/tree/master/kornia/filters. also the gaussian filtering in monai shows a possibility of handling 1d/2d/3d: https://github.com/Project-MONAI/MONAI/blob/master/monai/networks/layers/simplelayers.py#L79

Reply:
good idea!

Reply:
thanks for suggestions, we can create a separate `build_kernel` method for laplace kernel on different dimensions in the near future; padding equal to 1 has been fixed for this time being.